hacksite/hackapp/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Player(models.Model):
+	# Player inherits from Django's built-in user model
+	# The primary key for a player will be their username (phone number)
+	user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
So username will be based on phone number? What about email and name

Reply:
@mayasub